controllers/controllers.go

```diff
@@ -21,14 +21,27 @@
 package controllers
 
 import (
+	"fmt"
+	"time"
+
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	ctrl "sigs.k8s.io/controller-runtime"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
 var log = logf.Log.WithName("controller")
 
 const (
 	clusterFileKey = "cluster-file"
+
+	// podSchedulingDelayDuration determines how long we should delay a requeue
+	// of reconciliation when a pod is not fully scheduled.
```

Comment:
```suggestion
	// of reconciliation when a pod is not ready.
```

Or if ready is not the correct phrase we should drop the fully a Pod is either scheduled or not.